packages/design-docs/package.json

```diff
@@ -29,6 +29,7 @@
     "stylelint-config-standard": "^18.2.0"
   },
   "dependencies": {
+    "@inthepocket/itp-rcc-button": "^0.0.1",
```

Comment:
could you publish to npm before merging? currently you are using npm link to test I suppose